cns/service_test.go

```diff
@@ -322,3 +327,18 @@ func createTestCertificate(t *testing.T) string {
 
 	return testCertFilePath
 }
+
+func TestTLSVersionNumber(t *testing.T) {
```

Comment:
add test cases for unsupported: TLS 1.0, TLS 1.1 as per TLS version support comment above